templates/nginx.conf.j2

```diff
@@ -73,6 +73,10 @@ http {
     sendfile        on;
     #tcp_nopush     on;
 
+{% if nginx_main_template.http_settings.server_tokens is defined %}
```

Comment:
This will cover use cases were `server_tokens` is defined but it contains an empty value
```suggestion
{% if nginx_main_template.http_settings.server_tokens is defined and nginx_main_template.http_settings.server_tokens | length %}
```